AK/Math.h

```diff
@@ -67,11 +67,11 @@ constexpr T to_degrees(T radians)
 
 #define CONSTEXPR_STATE(function, args...)        \
     if (is_constant_evaluated()) {                \
-        if (IsSame<T, long double>)               \
+        if constexpr (IsSame<T, long double>)     \
             return __builtin_##function##l(args); \
-        if (IsSame<T, double>)                    \
+        if constexpr (IsSame<T, double>)          \
             return __builtin_##function(args);    \
-        if (IsSame<T, float>)                     \
+        if constexpr (IsSame<T, float>)           \
             return __builtin_##function##f(args); \
```

Comment:
This feels like it should be in a separate commit :)

Reply:
Hello :) it looks like the analyzer issue this was meant to resolve is no longer happening, and I have no local compile issues, so I've completely removed this from the patch.